microsoft-edge/hosting/WebView2.md

```diff
@@ -33,14 +33,29 @@ The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) reposit
 Help us build a richer WebView2 experience by sharing your feedback! Visit our [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports. It’s also a good place to search for known issues.
 During developer preview, we will also be collecting telemetry data to help us build a better WebView. Users can turn off WebView data collection by navigating to edge://settings/privacy in the browser and turning off browser data collection.
 
-## Evergreen vs. Bring-your-own
+## App Distribution
```

Comment:
Wondering if this should just be 'Distribution' instead of 'App Distribution'?  Most of it is about how the WebView bits get distributed in the context of the app vs the app itself.